spec/facility_generator_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'spec_helper'
+
+RSpec.describe FacilityGenerator do
+  before(:each) do
+    @fg = FacilityGenerator.new
+    @co = DmvDataService.new.co_dmv_office_locations
+    @ny = DmvDataService.new.ny_dmv_office_locations
+    @mo = DmvDataService.new.mo_dmv_office_locations
+    
+  end
+  describe '#initialize' do
+    it 'can initialize' do
+      expect(@fg).to be_an_instance_of(FacilityGenerator)
+    end
+  end
+
+  describe '#create_facility' do
+    it 'can create a facility' do
+      expect(@fg.create_facility(@co)).to be_an_instance_of(Array)
+    end
+  end
+  describe '#co_facility' do
+    it 'can create a CO facility and create a new instance of facility' do
+      
+      expect(@fg.co_facility(@co[0])).to be_an_instance_of(Facility)
+      expect(@fg.co_facility(@co[0]).name).to eq("DMV Tremont Branch")
+      expect(@fg.co_facility(@co[0]).address).to eq("2855 Tremont Place, Suite 118, Denver, CO 80205")
+      expect(@fg.co_facility(@co[0]).phone).to eq("(720) 865-4600")
+    end
```

Comment:
The only problem with testing for specific data, is that this data is coming from the api, we don't control it. So tomorrow, these tests might fail if CO adds a new facility tomorrow morning. We want our tests to test the functionality of our method and fail only if our method fails, not if the data changes. So we could instead test that each of these attributes is not nil so we know they're set to something.